packages/frontend/src/hooks/echarts/styles/tooltipStyles.ts

```diff
@@ -0,0 +1,165 @@
+import { type MantineTheme } from '@mantine/core';
+
+/**
+ * Helper to convert style object to CSS string
+ * We need this because ECharts doesn't support style objects directly
+ */
+const stylesToString = (styles: Record<string, string | number>) =>
+    Object.entries(styles)
+        .map(([key, value]) => {
+            // Convert camelCase to kebab-case
+            const kebabKey = key.replace(/([A-Z])/g, '-$1').toLowerCase();
+            return `${kebabKey}: ${value}`;
+        })
+        .join('; ');
+
+/**
+ * Get base tooltip styling
+ */
+export const getTooltipStyle = (theme: MantineTheme) => ({
+    padding: 8,
+    borderColor: theme.colors.gray[3],
+    borderWidth: 1,
+    borderRadius: 8,
+    backgroundColor: '#FFFFFF',
+    textStyle: {
+        color: theme.colors.gray[7],
+        fontSize: 12,
+    },
+    extraCssText:
+        'box-shadow: 0px 8px 8px 0px rgba(0, 0, 0, 0.08), 0px 0px 1px 0px rgba(0, 0, 0, 0.25);',
+});
+
+/**
+ * Format a tooltip value with pill styling (e.g. "100% - $100")
+ */
+export const formatTooltipValue = (
+    value: string,
+    theme: MantineTheme,
+): string => {
+    const styles = stylesToString({
+        display: 'inline-block',
+        backgroundColor: theme.colors.gray[0],
+        border: `1px solid ${theme.colors.gray[1]}`,
+        color: theme.colors.gray[7],
+        padding: '2px 8px',
+        borderRadius: '6px',
+        fontWeight: 600,
+        fontSize: '12px',
+    });
+    return `<span style="${styles}">${value}</span>`;
```

Comment:
do we need to sanitize `value` here? (is this something users can configure themselves in the UI? 🤔)